test/test_torch.py

```diff
@@ -7433,6 +7433,23 @@ def test_copy_broadcast(self, device) -> None:
         x.copy_(y)
         self.assertEqual(x[3], y)
 
+    def test_clamp(self, device):
+        x = torch.randn(100, 50)
```

Comment:
use make_tensor()

The current test is only being run on the CPU, but make_tensor() requires the device be specified (to help avoid this issue, which is an easy oversight that all of us have made at some point)